omero/sysadmins/unix/install-web.txt

```diff
@@ -199,3 +199,42 @@ The most popular configuration options include:
 
 -  Enabling a public user see :doc:`/sysadmins/public`.
 
+Setting up CORS
+---------------
+
+Cross Origin Resourse Sharing allows web applications hosted at other origins
+to access resources from your OMERO.web installation.
+This can be achieved using the
+`django-cors-headers <https://github.com/ottoyiu/django-cors-headers>`_ app
+with additional configuration of OMERO.web. See the
+`django-cors-headers <https://github.com/ottoyiu/django-cors-headers>`_ page
+for more details on the settings.
+
+Install the app and add it to the list of installed apps:
```

Comment:
in each of these examples, you can put the `::` at the end of the text rather than having : here and then the double ones below - it will render as : at the end of the line and then the code sample in the built docs and save these extra lines